NEWS.md

```diff
@@ -55,3 +55,5 @@
 
 * Documentation is now available also in *Dark* mode
   ([#2315](https://github.com/JuliaData/DataFrames.jl/pull/2315))
+* add rich display support for Markdown cell entries in HTML, Markdown and LaTeX
```

Comment:
what do you mean here by ", Markdown"? (we do not test shipping out data frame in Markdown anywhere - right?)